### PR TITLE
updating hapi with latest @hapi/hapi-18

### PR DESCRIPTION
**Pull request by scrajesh**
@hapi/hapi 18.3.2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "electrode-server",
-  "version": "2.2.2",
+  "version": "3.0.0",
```
Comment (jchip) [inline package.json]: please don't bump version

Comment (scrajesh) [reply package.json]: reverted the version bump

**Comment by jchip** [comment]
CI is actually broken but somehow reporting pass.

You changed a dep's name, there's no way that didn't break this module.

**Comment by jchip** [comment]
check CI, or run test locally.

https://travis-ci.org/electrode-io/electrode-server/jobs/583827676

**Comment by scrajesh** [comment]
`const Hapi = require("@hapi/hapi"); //should be the way to import Hapi instance from now.`

this resolved the build issue.

**Pull request merged**